app/views/layouts/application.html.haml

```diff
@@ -38,24 +37,19 @@
             %span.navbar-toggler-icon
           #navbarSupportedContent.collapse.navbar-collapse
             %ul.navbar-nav.ml-auto
+              %li.nav-item.active
+                = link_to t('.campaigns'), campaigns_path, class: 'nav-link'
               %li.nav-item.active
                 %a.nav-link{href: "#"}= t(".news")
               %li.nav-item.active
                 %a.nav-link{href: "#"}= t(".about_us")
-              %li.nav-item.dropdown
-                %a#navbarDropdown.nav-link.dropdown-toggle.active{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", href: "#", role: "button"}
-                  = t(".campaigns")
-                .dropdown-menu{"aria-labelledby" => "navbarDropdown"}
-                  %a.dropdown-item{href: "#"}= t(".something")
-                  %a.dropdown-item{href: "#"}= t(".something")
-                  %a.dropdown-item{href: "#"}= t(".something")
               %li.nav-item.active
                 %a.nav-link{href: "#"}= t(".contacts")
     %section.no-gutters
       .container
         - flash.each do |message_type, message|
-          .alert.alert-info
-            %button.close{'data-dismiss' => 'alert', type: 'button'} ×
+          %div{class: "alert flash_message alert-#{message_type == 'notice' ? 'success' : 'info'}"}
```

Comment:
When you add a message to your flash object you can use any key you want. It works as a usual hash, there are no restrictions on keys, not just `notice` or `alert` are available.

You can add a `success` of `info` message like this` flash[:success]` or `flash[:info]`

https://stackoverflow.com/questions/9063751/what-flash-message-types-are-available-in-rails
http://api.rubyonrails.org/classes/ActionDispatch/Flash/FlashHash.html